carma-messenger-core/message/.gradle/2.14.1/taskArtifacts/cache.properties

```diff
@@ -0,0 +1 @@
+#Fri Jul 24 06:57:58 PDT 2020
```

Comment:
Let's also make sure all these build artifacts are cleaned up as well, only source code should be committed.